example/propagation/federatedjob-placement.yaml

```diff
@@ -5,5 +5,4 @@ metadata:
   namespace: test-namespace
 spec:
   clusterNames:
-  - cluster2
-  - cluster1
+  - "null"
```

Comment:
Consider using a more specific replacement token such as `CLUSTER_NAME` to more clearly indicate that replacement is expected.